inst/@sym/sym.m

```diff
@@ -277,7 +277,6 @@
   end
 
   asm = {};
-  check = true;
```

Comment:
I'm not sure I understand.  Is this about `sym` explicitly calling itself?  I thought we were not supposed to have the sym ctor *explicitly* calling `sym(...)`, but it was ok if sym called some other function which then called `sym(...)`.

What do you mean by "extra lines script every time we call that function for the limitations"?  Can you give me an explicit bit of code that will need to change?

Sorry, but I've kind of lost track of the various classdef issues we face.

Does this comment need to be addressed before I merge this?